frontend/appflowy_flutter/lib/plugins/database_view/board/presentation/board_page.dart

```diff
@@ -295,6 +298,7 @@ class _BoardContentState extends State<BoardContent> {
         rowCache: rowCache,
         cardData: groupData.group.groupId,
         groupingFieldId: groupItem.fieldInfo.id,
+        groupId: groupData.group.groupId,
```

Comment:
This looks a bit weird to me:
```
cardData: groupData.group.groupId,
groupId: groupData.group.groupId,
```

Reply:
Great catch, thanks a bunch. Let's just use the `cardData` since it's there. That field was originally already meant to carry custom information.

Reply:
Hi @Xazin @richardshiue this was intentional as the cardData field seems to be used for different purpose [here in calendar_day](https://github.com/AppFlowy-IO/AppFlowy/blob/main/frontend/appflowy_flutter/lib/plugins/database_view/calendar/presentation/calendar_day.dart#L309) to avoid complications I created a new field. Please let me know if my understanding isn't correct

Reply:
For now, we can leave it as is.